controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -375,6 +375,26 @@ bool master_thread::handle_cmdu_1905_autoconfiguration_search(Socket *sd,
     return son_actions::send_cmdu_to_agent(sd, cmdu_tx);
 }
 
+/**
+ * @brief Check if the operating class of @a radio_basic_caps match the operating classes of
```

Comment:
What is "@a" ?

Reply:
@tomereli, try to read the brief in line 379, you will understand what I meant.

Reply:
@a is a command to format the following word as code. It is meant to be used for arguments to functions, since those are not auto-formatted (unlike class and function references).

Cfr. http://doxygen.nl/manual/commands.html#cmda